### DISCUSSION TITLE
Issues when applying to Github Student

### DISCUSSION THREAD

**Discussion by GCWebLabs** [community/community#58363] [GitHub Education]
### Hi everyone,

Hi Github community, I have to cope with issues making nonsense when applying to Github students. I'm French so I sometimes don't really understand the issues. Here are the problems :

- My profile is completed but i'm told to add my full name in my profile.
- My location doesn't correspond to the one of my high school, and the Github chatbot told me that I should apply from a computer of my school. But there's no webcam on its PC and I can't apply from elsewhere, otherwise I apparently am not at the same place as my school (I'm in the same city)
- The text is apparently unable to be read, or I have to translate it. But how can I translate an official document ? By putting another paper besides close to the French one ?
- I am told there's not the name of my college on the document, but there is ! I don't understand
- There should be a date on the document, but in the OFFICIAL document in France to attest that I'm in that school, there's only the date of the publication of the document.

Please help me guys, I did like, 10 appliances, and there are more and more errors every time, I can't bear it ! In addition to that it's soon the end of the year and I really need Github Students to work hard during the summer vacation !

**Comment by the-officialjosh** [comment]
Let's try to address these issues one by one in order to find solutions:

Adding your full name: It is important to ensure that you add your full name exactly as it appears on your document, using sentence case. This will help maintain consistency and accuracy throughout the application process.

School location requirement: To comply with the guidelines and ensure the integrity of the process, it is necessary to be physically present at your school location when completing the application. This helps prevent misuse of email addresses and ensures that the platform is accessed only by the intended individuals.

Document translations: Typically, translations are not required for official documents. It is recommended to focus on proper document placement and clarity. Adding additional papers for translation purposes may not be necessary and could potentially cause confusion. Instead, try using your examination or semester results as they might address the requirements. If your results are accessible through a browser, you can utilize translation tools within the browser to assist you.

These suggestions are based on my understanding, and I'm here to help explore additional possibilities to resolve your problem. Please let me know if any of these worked.

**Reply by GCWebLabs** [reply]
Sorry for answering that late, i've been away for some time!
Thank you for your answer, i'll try to do that :)
